ffi/tests/ctests/src/tests.c

```diff
@@ -0,0 +1,307 @@
+#include <stdlib.h>
```

Comment:
This file uses tabs, while Rust files use 4 spaces indentation (due to Rustfmt). Let's stick to the same here for consistency.

Reply:
I'll fix that asap. I don't know what happened to my editor